packages/superset-ui-plugins-demo/tsconfig.json

```diff
@@ -1,23 +1,28 @@
 {
```

Comment:
is this file not generated by beemo?

Reply:
Hmm... true. The one in `demo` is somehow not generated and is also checked in.